app_flutter/lib/agent_health_details.dart

```diff
@@ -70,6 +63,17 @@ class AgentHealthDetails {
   /// The agent to show health details from.
   final Agent agent;
 
+  /// The date and time of the last time this agent pinged Cocoon to show it is
+  /// still responsive and able to take tasks.
+  ///
+  /// Agents that become unresponsive tend to be unresponsive because a process
```

Comment:
nitty nit: there's a lot of "unresponsive" in this sentence

Reply:
i mean, you're not wrong...

fixed.